app/Transformers/TextTransformer.php

```diff
@@ -92,9 +92,9 @@ public function transformForV2($text)
              *   @OA\Xml(name="v2_text_verse"),
              *   @OA\Items(
              *              @OA\Property(property="book_name",         ref="#/components/schemas/Book/properties/name"),
-             *              @OA\Property(property="book_id",           ref="#/components/schemas/Book/properties/id_osis"),
+             *              @OA\Property(property="book_id",           ref="#/components/schemas/Book/properties/name"),
```

Comment:
This property should be the id of the book I think, because name is the English name of the book
![image](https://user-images.githubusercontent.com/41348080/95923452-281c1680-0d7b-11eb-9ff8-e2e150647500.png)

Reply:
The objective for the v2 endpoints is to match what DBP2 returns. In this case, DBP4 introduced a change related to book id. In DBP2, the book id is similar to "Luke", but in DBP4, the id is like "LUK". So I had to pick book.name to exactly match what DBP2 returned. Note that this only applies to v2 endpoints.  Is there another field that we should return that will provide the DBP2 version of the bookname?
For reference, here is the object currently returned by DBP4
![Screen Shot 2020-10-13 at 6 54 38 PM](https://user-images.githubusercontent.com/1403755/95928048-0b391080-0d86-11eb-8627-eac4e9672684.png)